### ISSUE TITLE
Kernel panic on reboot after post installation

### ISSUE THREAD

**Issue by stingray88** [theracermaster/Gigabyte-GA-Z77X-macOS-Install#17]
Z77X-UD5H - 3770K - 32GB RAM - GTX 980 Ti

So I followed the guide, and through a bit of trouble I was finally able to get 10.11.1 installed. However when it came time to reboot after the post installation scripts were run (step 3) I got a kernel panic on boot. Attached is a screenshot.

![img_0050](https://cloud.githubusercontent.com/assets/1384484/10720355/6b1b7ada-7b53-11e5-8cf5-3ead61beff85.JPG)

 I tried booting in safe mode, without caches and kext injection, with cpus=1 and with PEG and IGPU. Same story either way. If you've got any suggestions for me to try, I'm all ears... thanks!

**Comment by theracermaster** [comment]
Please make sure you are using the absolute latest version from the repo.

**Comment by stingray88** [comment]
I am using 1.8.4 with Clover 3302

**Comment by theracermaster** [comment]
Just to clarify - booting from the installer USB worked fine? Only booting after post-install doesn't work?

**Comment by stingray88** [comment]
Yes. After step 3 here - https://github.com/theracermaster/Gigabyte-GA-Z77X-DSDT-Patch/wiki/3.-Post-Installation

I ran all of the terminal commands listed, and then rebooted. After reboot this is the kernel panic I get.

**Comment by theracermaster** [comment]
See if deleting SSDT-HACK.aml lets you boot. If it does, then it's an issue with the SSDT, but one I haven't seen (*yet*). However, I'm not really sure why this is happening, the graphics injection in the installer config is the same as the post-installation config.

**Comment by stingray88** [comment]
Sorry it's taken so long to get back to you about this. I got really busy with a couple of editing projects and had to role back to 10.10.5 on my main drive from a backup so that I could get back to work. I'll have to test this again in the future and hopefully get better results.

To answer your questions though, I did install OS X with my 980 Ti installed. I did not try with it removed. Also I had PEG set for Init Display First, as stated in the UEFI config guide you have here.

Sometime in the future when I have more time I'd like to test your method of installation again, however I unfortunately don't have the time right now. I will report back with my results at that time, as I want to further help make your scripts as robust as possible for all of us!
